db/migrate/20160329212403_add_switch_uuid_and_is_shared_to_switch.rb

```diff
@@ -0,0 +1,6 @@
+class AddSwitchUuidAndIsSharedToSwitch < ActiveRecord::Migration[5.0]
+  def change
+    add_column :switches, :switch_uuid, :string
+    add_column :switches, :is_shared, :boolean, :default => false
```

Comment:
Prefer `shared` over `is_shared`.  Also, remove the default option...a column's default is nil which is already falsey, and even so we should prefer using default_value_for if it's really necessary.